src/csharp/Microsoft.Spark/Sql/SparkSession.cs

```diff
@@ -61,10 +61,39 @@ internal SparkSession(JvmObjectReference jvmObject)
         /// <returns>Builder object</returns>
         public static Builder Builder() => new Builder();
 
-        /// Note that *ActiveSession() APIs are not exposed because these APIs work with a
-        /// thread-local variable, which stores the session variable. Since the Netty server
-        /// that handles the requests is multi-threaded, any thread can invoke these APIs,
-        /// resulting in unexpected behaviors if different threads are used.
+        /// <summary>
+        /// Changes the SparkSession that will be returned in this thread and its children when
+        /// SparkSession.GetOrCreate() is called. This can be used to ensure that a given thread
+        /// receives a SparkSession with an isolated session, instead of the global (first created)
+        /// context.
```

Comment:
I don't think SparkSession.GetOrCreate exists.  Probably referring to the builder. 

```suggestion
        /// Changes the SparkSession that will be returned in this thread and its children when
        /// <see cref="Builder.GetOrCreate"/> is called. This can be used to ensure that a given
        /// thread receives a SparkSession with an isolated session, instead of the global
        /// (first created) context.
```